doc/Financial.md

```diff
@@ -0,0 +1,105 @@
+# How to run the SPA with the Financial-grade Token Handler
+
+## Overview
+
+The token handler uses the following behaviour:
+
+- OAuth work is done for the SPA by an API
+- Authorization Code Flow (PKCE) is used, along with a Mutual TLS client secret
```

Comment:
I would say `Authorization Code Flow with PKCE is used, along with a Mutual TLS client authentication`